src/css/modules/pages/home.css

```diff
@@ -1,18 +1,19 @@
 .hero--home .hero__title {
   --max-size: var(--font-size-9);
-  text-align: center;
 }
 
 .hero--home .hero__body {
   font-size: var(--font-size-6);
-  margin-left: auto;
-  margin-right: auto;
-  text-align: center;
 }
 
-.home-model .content-block__intro,
-.home-portfolio .content-block__intro {
-  margin-left: auto;
+/* Should I be using em or px? */
```

Comment:
@will0101 I always us `em` for media queries. I do that based on the findings in this article https://zellwk.com/blog/media-query-units/

Reply:
There's a lot of great stuff in this article. I just bookmarked it for future reference.